easybuild/easyconfigs/f/FLANN/FLANN-1.8.4-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,45 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'FLANN'
+version = '1.8.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/mariusmuja/flann/'
+description = "FLANN is a library for performing fast approximate nearest neighbor searches in high dimensional spaces."
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['https://github.com/mariusmuja/flann/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = [
+    'ed5843113150b3d6bc4c325fecb51337838a9fc09ad64bdb6aea79d6e610ee13',  # flann-1.8.4.tar.gz
+    '662e5972487a6616cbca36ac9459bcc3ca773ed781564ed2532d9792d97c2897',  # cmake-flann-1.8.4.patch'
+]
+
+builddependencies = [('CMake', '3.16.4')]
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', '-Python-3.8.2'),
+]
+
+separate_build_dir = True
+
+patches = ['%(name)s-%(version)s_fix-cmake.patch']
```

Comment:
please move this line to right below `sources`

Reply:
Done. This also means your earlier comment about the patch file is superseded by this one here.